src/state/routing/types.ts

```diff
@@ -68,20 +67,20 @@ export interface GetQuoteResult {
   routeString: string
 }
 
-export class InterfaceTrade<
-  TInput extends Currency,
-  TOutput extends Currency,
-  TTradeType extends TradeType
-> extends Trade<TInput, TOutput, TTradeType> {
-  gasUseEstimateUSD: CurrencyAmount<Token> | null | undefined
+class ClassicTrade<TInput extends Currency, TOutput extends Currency, TTradeType extends TradeType> extends Trade<
```

Comment:
Is this used anywhere? Is there any reason it can't be called `InterfaceTrade`, and extend a concrete type of `Trade`, instead of re-aliasing it just underneath?

Reply:
it is now used in one place!